src/lib/Libcmds/pbs_json.c

```diff
@@ -224,6 +263,30 @@ free_json_node_list()
 	prev_link = NULL;
 }
 
+/**
+ * @brief
+ *	Determines if 'str' contains only white-space characters.
+ *
+ * @param[in] str - string to check
+ *
+ * @return	int
+ * @retval	1	if 'str' contains only white-space characters.
+ * @retval	0	error or otherwise.
+ *
+ */
+static int
+whitespace_only(char *str) {
```

Comment:
can you make this `const char *str`

Reply:
Sure

Reply:
@vstumpf : I've made the change. I need a re-review and feel free to merge if this PR is acceptable to you. Thanks.